docs/tutorials/using_apis/Mbed_USB_WAV_Audio_Player/Mbed_USB_WAV_Audio_Player.md

```diff
@@ -0,0 +1,88 @@
+# Mbed USB WAV Audio Player
+This is a tutorial on how to put together a USB WAV audio player with Mbed. In general, the user will load a WAV file onto an SD card that will be connected to an Mbed board. The user can then play the WAV file from the board on a host computer over USB. Once connected, the Mbed board will act as a USB microphone that will be playing the sound, so that the sound can be looped back to speakers, headphones, or recorded with a program like Audacity.
+
+### Libraries
+This tutorial uses several Mbed libraries:
+- USB (part of Mbed-OS)
+- [sd-driver](https://github.com/ARMmbed/sd-driver) (separate Mbed library)
+- [AudioPlayer](https://github.com/c1728p9/AudioPlayer) (separate Mbed library)
+
+The USBAudio class of USB library is used to send audio data from the Mbed board to the host PC. The SDBlockDevice and FATFileSystem classes of sd-driver library are used so that WAV files can be stored on an SD card and accessed by the Mbed board. Lastly, the AudioPlayer and WaveAudioStream classes of the AudioPlayer library are used to access the audio data from the WAV file on the SD card.
+
+### What You Need
+- Mbed Enabled board
+- SD card
+- SD card module (if board does not already have one)
+- USB cable
+
+### Setup
+The following steps demonstrate the setup and use of the Mbed WAV Audio Player:
+1. Create a new Mbed project.   
+`mbed new <project name>`
+2. Add the [sd-driver](https://github.com/ARMmbed/sd-driver) and [AudioPlayer](https://github.com/c1728p9/AudioPlayer) libraries into the new Mbed project directory.   
+`mbed add https://github.com/ARMmbed/sd-driver`    
+`mbed add https://github.com/c1728p9/AudioPlayer`
+3. Copy and paste the below example code into *main.cpp*.
```

Comment:
It may be good to have another step to update SPI pins of `SDBlockDevice` for your board.